storage/expanding-persistent-volumes.adoc

```diff
@@ -21,3 +21,14 @@ include::modules/storage-expanding-filesystem-pvc.adoc[leveloffset=+1]
 
 include::modules/storage-expanding-recovering-failure.adoc[leveloffset=+1]
 endif::openshift-enterprise,openshift-webscale,openshift-origin[]
+
+[role="_additional-resources"]
+.Additional resources 
+
+ifndef::microshift[]
```

Comment:
So this is the OCP storage assembly (not something that is shared by both).

So this one should just have this first bullet xref and no conditionals. And then you'd go to the microshift assembly and add the microshift xref.